bom/pom.xml

```diff
@@ -692,7 +692,7 @@
 			<dependency>
 				<groupId>com.gentics.graphqlfilter</groupId>
 				<artifactId>graphql-java-filter</artifactId>
-				<version>2.0.0</version>
+				<version>2.0.1-SNAPSHOT</version>
```

Comment:
TODO fix to release when ready.
https://github.com/gentics/graphql-java-filter/pull/5

Reply:
Should it be `2.1.0` / `3.0.0` ?